.github/workflows/release.yml

```diff
@@ -391,6 +391,7 @@ jobs:
           && !inputs.prerelease
           && !github.events.inputs.test_release
           && contains(env.PAUL_BLART, github.actor)
+          && github.event_name != 'schedule'
```

Comment:
this is unnecessary, we're already checking that the `event_name` is `workflow_dispatch`